### PR TITLE
Add iquilezles.org

### PR DESCRIPTION
**Pull request by ShadowMitia**

### CODE REVIEW

File: resources/index.html [hunk 1]
```diff
@@ -84,6 +84,7 @@
 		* [A Comparison of Modern Graphics APIs](https://alain.xyz/blog/comparison-of-modern-graphics-apis)
 		* [Scratchapixel](https://www.scratchapixel.com)
 		* [The Graphics Codex](https://graphicscodex.com/)
+		* [iquilezles.org](https://iquilezles.org/)
```
Comment (deccer) [inline resources/index.html]: I would like to have a bit more descriptive description here

**Review by deccer — CHANGES_REQUESTED** [review]
Smol thing

**Comment by ShadowMitia** [comment]
I tried keeping it all inside the brackets. I'm open to suggestions on improving it, but I might have to add some description outside of the markdown link if that's ok?

File: resources/index.html [hunk 2]
```diff
@@ -84,7 +84,7 @@
 		* [A Comparison of Modern Graphics APIs](https://alain.xyz/blog/comparison-of-modern-graphics-apis)
 		* [Scratchapixel](https://www.scratchapixel.com)
 		* [The Graphics Codex](https://graphicscodex.com/)
-		* [iquilezles.org](https://iquilezles.org/)
+		* [Inigo Quilez's articles and tutorials (and more)](https://iquilezles.org/)
```
Comment (deccer) [inline resources/index.html]: Hmm why not link to specific articles of his, in various sections... like we have an ibl section, so link the ibl articles..... SDF section, link the sdf articles etc...[Link-to-Link](link) is a little thin

**Review by deccer — CHANGES_REQUESTED** [review]

File: resources/index.html (hunk 2 above)
Comment (ShadowMitia) [reply resources/index.html]: That's a lot of sections :sweat_smile: But if you'd prefer that I'll make the changes.

Comment (deccer) [reply resources/index.html]: I mean i understand why you just linked iquilezles as a whole :) if you are on our discord, we could discuss this thing on how to tackle those kind of resources

**Review by deccer — APPROVED** [review]
LGTM

**Pull request merged**